tests/conftest.py

```diff
@@ -220,6 +220,44 @@ def _create_profile(name, **kwargs):
     return _create_profile
 
 
+@pytest.fixture
+def create_config_instance(create_empty_config_instance, create_profile) -> Config:
```

Comment:
The naming is a bit confusing, since some of these fixtures (`create_config_instance`) are factory-type fixtures (return a function) while others (`create_empty_config_instance`) return the instance.

If we keep the current implementation, I'd suggest something like

 * `create_empty_config_instance` => `empty_config`
 * `create_config_instance` => `config_with_profile_factory`
 * `create_profile` => `profile_factory`

Reply:
changed in ac172c4b8df88df4b8bdd4ba4baa1e7cb4959484